miner/algo_common.go

```diff
@@ -262,14 +262,31 @@ func (envDiff *environmentDiff) commitBundle(bundle *types.SimulatedBundle, chDa
 	bundleSimEffGP := new(big.Int).Set(bundle.MevGasPrice)
 
 	// allow >-1% divergence
-	bundleActualEffGP.Mul(bundleActualEffGP, big.NewInt(100))
+	bundleActualEffGP.Mul(bundleActualEffGP, common.Big100)
 	bundleSimEffGP.Mul(bundleSimEffGP, big.NewInt(99))
 
 	if bundleSimEffGP.Cmp(bundleActualEffGP) == 1 {
 		log.Trace("Bundle underpays after inclusion", "bundle", bundle.OriginalBundle.Hash)
 		return errors.New("bundle underpays")
 	}
 
+	if enforceProfit {
+		// if profit is enforced between simulation and actual commit, only allow >-1% divergence
+		simulatedBundleProfit := new(big.Int).Set(bundle.TotalEth)
+		actualBundleGasFees := new(big.Int).Mul(bundleActualEffGP, big.NewInt(int64(gasUsed)))
+		actualBundleProfit := new(big.Int).Add(coinbaseBalanceDelta, actualBundleGasFees)
```

Comment:
its still looks incorrect. coinbaseBalanceDelta is actual bundle profit

Reply:
fixed, discussed offline